react/features/analytics/handlers/amplitude/fixDeviceID.native.ts

```diff
@@ -1,6 +1,9 @@
 import { Amplitude } from '@amplitude/react-native';
 import DefaultPreference from 'react-native-default-preference';
-import DeviceInfo from 'react-native-device-info';
+import { getUniqueId } from 'react-native-device-info';
```

Comment:
No need to use the destructured import, but make sure the API doesn't return a Promixe, or await it is we are in an async context.